compiler-core/src/parse/error.rs

```diff
@@ -308,6 +308,21 @@ utf16_codepoint, utf32_codepoint, signed, unsigned, big, little, native, size, u
                     "See: https://tour.gleam.run/flow-control/case-expressions/".into(),
                 ],
             ),
+            ParseErrorType::UnknownAttributeRecordConstructor => (
+                "Variant attribute(s) are unknow.",
+                vec!["Try `@deprecated`.".into()],
+            ),
```

Comment:
I think I'd rather have an error that explicitly says that the attribute cannot be used on a variant for each of the attributes rather than a single error for all of those.

```
pub type Wibble {
  @external(javascript, "", "")
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ This attribute cannot be used on a type variant
  Wobble
}
```

Also I would not include that hint, it doesn't make much sense to suggest using `deprecated` if I wrote an `@external` or `@internal` attribute